src/ServicePulse.Host/app/js/app.constants.js

```diff
@@ -5,7 +5,7 @@
         .constant('showPendingRetry', false)
         .constant('scConfig', {
             service_control_url: 'http://localhost:33333/api/',
-            monitoring_urls: ['http://localhost:33633/']
+            monitoring_urls: ['http://localhost:1234/']
```

Comment:
now the question is do we want to, as SC.Monitoring while being run from VS use 1234 as a port